.github/workflows/npm-publish.yml

```
@@ -0,0 +1,28 @@
+# This workflow will run tests using node and then publish a package to NPM when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Publish to NPM
+
+on:
+  release:
+    # Run this on both stable and pre-release
+    types: [published]
+
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # Check out the referred tag in the release
+          # In case we want to publish somethign other than HEAD
+          ref: $GITHUB_SHA
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: ".nvmrc"
+          registry-url: https://registry.npmjs.org/
+          scope: "@ftrack"
+      - run: yarn install --immutable
+      - run: yarn publish --access public
```

I believe this should be:
```suggestion
      run: yarn npm publish --access public
```